features/old/services/multiservice.feature

```diff
@@ -15,6 +15,7 @@ Feature: Multiservice feature
     When I am on the provider dashboard
     Then I should see "New API"
 
+  @javascript
```

Comment:
@guicassolato to fix this test I just added the @javascript tag, but maybe you know a better way